example_projects/transfer/nlb_observability_stack/variables.tf

```diff
@@ -0,0 +1,153 @@
+# AWS resource needed variables
+variable "tags" {
+  type = map(string)
+  default = {
+    Example = "test"
+  }
+  description = "Tags for AWS resources"
+}
+
+variable "bucket_name" {
+  type        = string
+  default     = "double-cloud-nlb-logs-bucket"
+  description = "Name of the sample bucket where the NLB logs should be stored"
+}
+
+variable "bucket_prefix" {
+  type        = string
+  default     = "metrics"
+  description = "Bucket prefix for NLB logs in S3 bucket"
+}
+
+variable "nlb_name" {
+  type        = string
+  default     = "preprod-metric-exp-dt-tls"
```

Comment:
fix name or remove at all

Reply:
please clarify whats there to fix in an example name ?

Reply:
removed variable